packages/sources/svelte/src/components/selection-controls/VtmnCheckbox/VtmnCheckbox.svelte

```diff
@@ -0,0 +1,42 @@
+<script>
+  import { cn } from '../../../utils/classnames';
+
+  /**
+   * ID of the checkbox
+   * @type {string}
+   */
+  export let id;
+
+  /**
+   * Label of the checkbox
+   * @type {string}
+   */
+  export let label;
+
+  /**
+   * Whether the checkbox is disabled
+   * @type {boolean}
+   */
+  export let disabled;
+
```

Comment:
Need to add a `export let checked` to provide a default state to the checkbox

Reply:
Can't be passed through the `$$restProps` ?

Reply:
Right ! It can be passed through `$$restProps` but it is a reactive property which is interact directly with the component.
For me, with this case, the property must be exported.

Reply:
ok